code/modules/antagonists/changeling/abilities/regeneration.dm

```diff
@@ -33,6 +33,7 @@
 				implants += I
 
 			H.in_fakedeath = 1
+			APPLY_MOB_PROPERTY(C, PROP_CANTMOVE, "fakedeath")
```

Comment:
Should this and regeneration.dm:73 be using the same index or whatever? One uses src.type, one uses "fakedeath"

Reply:
oh yes definitely